doc/source/conf.py

```diff
@@ -1,174 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Again, try running `make docs` and see if it builds and launches the docs on your local machine but this file (or parts of it) may need to be copied to our conf.py